data/Makefile.am

```diff
@@ -51,14 +50,12 @@ CLEANFILES =						\
 	gnome-initial-setup.desktop			\
 	gnome-initial-setup-copy-worker.desktop.in	\
 	gnome-initial-setup-copy-worker.desktop		\
-	gnome-initial-setup-first-login.desktop		\
 	com.endlessm.TestMode.policy			\
 	$(NULL)
 
 autostartdir = $(sysconfdir)/xdg/autostart
 autostart_DATA =					\
 	gnome-initial-setup-copy-worker.desktop		\
-	gnome-initial-setup-first-login.desktop		\
```

Comment:
@mcatanzaro I think just avoiding the installation of the desktop file is a good approach, but I think the only change that's needed in that case is the removal of this line.
I actually would prefer to comment it out with a note like you propose.